packages/wallet/deployment/deploy-test.js

```diff
@@ -0,0 +1,49 @@
+const ethers = require('ethers');
+const etherlime = require('etherlime-lib');
+const path = require('path');
+const writeJsonFile = require('write-json-file');
+const loadJsonFile = require('load-json-file');
+
+const trivialAppArtifact = require('../build/contracts/TrivialApp.json');
+
+let contractsToAddresses = {};
+
+async function getNetworkMap() {
+  try {
+    return await loadJsonFile(path.join(__dirname, 'network-map.json'));
+  } catch (err) {
+    if (!!err.message.match('ENOENT: no such file or directory')) {
+      return {};
```

Comment:
Maybe we should add a single `console.warn` to indicate that this file was not found?

Reply:
```suggestion
      console.warn("network-map.json not found");
      return {};
```